2-multitenant/modules/env_baseline/outputs.tf

```diff
@@ -51,3 +51,10 @@ output "ip_address_self_links" {
     for value in module.ip_address : value.self_links
   ]
 }
+
+output "ip_addresses" {
+  description = "IP Addresses"
+  value = [
+    for value in module.ip_address : value.addresses
```

Comment:
As this ip_address has a name, might want to use `value.name` => `value.address` array to allow for additional in the future?

Reply:
https://github.com/terraform-google-modules/terraform-google-address/blob/master/outputs.tf

names and addresses are separate output, may be leave out names for later when needed?

Reply:
I was thinking an array so it could be retrieved by key if more were added, but your call.  Leaving as-is is fine.